tests/src/test/scala/neotypes/BaseIntegrationSpec.scala

```diff
@@ -13,7 +13,7 @@ import scala.concurrent.duration._
 import scala.jdk.DurationConverters._
 
 /** Marker trait to signal that the test needs a driver. */
-trait DriverProvider[F[_]] extends { self: BaseIntegrationSpec[F] =>
+trait DriverProvider[F[_]] extends AnyRef { self: BaseIntegrationSpec[F] =>
```

Comment:
Scala 3 doesn't allow `extends` without something class-like.

Reply:
Oh, that `extends` was wrong all along, just remove it.